docs/extensions/toc.md

```diff
@@ -174,6 +174,10 @@ The following options are provided to configure the output:
 * **`permalink_title`**:
     Title attribute of the permanent link. Defaults to `Permanent link`.
 
+* **`permalink_prepend`**:
+    Set to `True` if the generated permalinks should be placed at the
```

Comment:
Note that the spellchecker is failing on `permalink` when the word is not in a code span (code is ignored by the spellchecker). Obviously, the word `permalink` is not in the dictionary. While we could add it, we usually write our documentation with formal language which uses real words only and reserve the short names for code only. If you notice, the other options use "permanent link" in their descriptions, rather than `permalink`.